app/src/main/java/com/github/swent/echo/compose/event/EventTagEntry.kt

```diff
@@ -35,9 +42,28 @@ fun EventTagEntry(
     onTagSelected: (tag: Tag) -> Unit,
     onTagDeleted: (tag: Tag) -> Unit
 ) {
-    var dialogVisible by remember { mutableStateOf(false) }
     Column(modifier = Modifier.fillMaxWidth().padding(EVENT_PADDING_BETWEEN_INPUTS)) {
         EventEntryName(stringResource(R.string.edit_event_screen_tags))
+        Row {
```

Comment:
Some repetitions here, you could create a list of Pair<String, String> and then iterates over it

Reply:
Thanks a lot for the suggestions, I changed that.